src/org/opensolaris/opengrok/web/WebappListener.java

```diff
@@ -65,15 +65,17 @@
     public void contextInitialized(final ServletContextEvent servletContextEvent) {
         ServletContext context = servletContextEvent.getServletContext();
         RuntimeEnvironment env = RuntimeEnvironment.getInstance();
-
+        
         String config = context.getInitParameter("CONFIGURATION");
+        File configFile = new File(config);
+               
         if (config == null) {
             LOGGER.severe("CONFIGURATION section missing in web.xml");
         } else {
             try {
-                env.readConfiguration(new File(config));
+                env.readConfiguration(configFile);
             } catch (IOException ex) {
-                LOGGER.log(Level.WARNING, "OpenGrok Configuration error. Failed to read config file: ", ex);
+                LOGGER.log(Level.WARNING, "OpenGrok Configuration error. Failed to read config file: {0}", ex.toString());
```

Comment:
Log levels are Warning for missing config, so exception is not thrown, but message is written to logger